metric-example/metric-expressions-for-k8s.md

```diff
@@ -0,0 +1,206 @@
+# Metric Expressions for K8s
+This document collects frequently asked metric expressions for typical K8s use-cases. Listed metric expressions can be used for [charting](https://www.dynatrace.com/support/help/how-to-use-dynatrace/dashboards-and-charts/explorer/explorer-advanced-query-editor/) or alerting using [custom metric events](https://www.dynatrace.com/support/help/how-to-use-dynatrace/problem-detection-and-analysis/problem-detection/metric-events-for-alerting/advanced-metric-queries/).
+
+**Table of contents**
+
+<!-- TOC -->
+
+- [Prerequisites](#prerequisites)
+- [Important note](#important-note)
+- [Nodes utilization](#nodes-utilization)
+    - [CPU](#cpu)
+    - [Memory](#memory)
+- [Node conditions](#node-conditions)
+- [Workloads health](#workloads-health)
+    - [Not all pods running](#not-all-pods-running)
+    - [Not all containers running](#not-all-containers-running)
+- [Workloads resource utilization and optimization](#workloads-resource-utilization-and-optimization)
+    - [Slack](#slack)
+    - [Usage above requests](#usage-above-requests)
+    - [Usage in terms of limits](#usage-in-terms-of-limits)
+    - [High trotthling](#high-trotthling)
+    - [Memory](#memory)
+    - [High container restart rate](#high-container-restart-rate)
+- [Further Reading](#further-reading)
+
+<!-- /TOC -->
+
+## Prerequisites
+This document assumes basic familiarity with metric selectors. If you don't yet feel comfortable with simple metric selectors, consider reading
+through [Query by Example](query-by-example.md) first.
+
+You can try the examples listed here through the Web UI via _Data Explorer_,
+or you can send selectors to the `/v2/metrics/query` API for evaluation.
+
+## Important note
+**Depending on the size of your environment, the following examples might put extrem load on your Dynatrace environment. Please use with care and try using filters whenever possible to narrow the scope of these queries.**
+
+[//]: # (For Dynatrace employees: A dashboard containing all these metric expressions can be found here: https://stratus.dev.dynatracelabs.com/#dashboard;gtf=-2h;gf=all;id=64bfca68-49ec-4628-92d2-ab38d0cf28f7)
```

Comment:
What is this syntax for? It seems this part is invisible 🤔 
https://github.com/floriangeigl/dynatrace-api/blob/master/metric-example/metric-expressions-for-k8s.md

Reply:
yes - it's a comment in markdown.